Revised RHV permissions (#491)
kubev2v/forklift-documentation: documentation/modules/rhv-prerequisites.adoc

The following prerequisites apply to {rhv-full} migrations:

* To create a provider, you must have at least the `UserRole` and `ReadOnlyAdmin` roles assigned to you. These are the minimum required permissions, however, any other administrator or superuser permissions will also work.

> * To migrate virtual machines:

Reviewer: Looking good overall. The only thing I'm not sure about is if it's clear enough that both sets of permissions had to be given together. The first one is required for the addition of the provider, but it's also important to leave them after for the migration itself.

Author: @bkhizgiy I don't understand. Do you need to have `UserRole` and `ReadOnlyAdmin` roles as well as RHV admin permissions (and  `DiskCreator` and `UserVmManager` permissions)  to migrate VMs?

Reviewer: @RichardHoch I think I didn't explain myself well; sorry for the confusion.

The first set of permissions, `UserRole` and `ReadOnlyAdmin`, are required for the addition of the provider but also need to remain for the migration itself. That means if the user adds these permissions, then adds the provider, and afterward removes the permissions, the migration will fail.

The second set of permissions is required only for the migration and needs to be granted only to the VMs the user wishes to migrate. However, it should be given in addition to the first set of permissions.

I just thought it might not be clear enough from the current phrasing.

The following prerequisites apply to {rhv-full} migrations:

* To create a source provider, you must have at least the `UserRole` and `ReadOnlyAdmin` roles assigned to you. These are the minimum required permissions, however, any other administrator or superuser permissions will also work.

[IMPORTANT]
====
You must keep the `UserRole` and `ReadOnlyAdmin` roles until the virtual machines of the source provider have been migrated. Otherwise, the migration will fail.
====

* To migrate virtual machines:
** You must have one of the following:
*** {rhv-short} admin permissions. These permissions allow you to migrate any virtual machine in the system.

> *** `DiskCreator` and `UserVmManager` permissions. These permissions have to be granted to you individually by an `admin` user per each individual virtual machine that you want to migrate.

Reviewer 2: I think the second part of this sentence is too detailed - it would be ok if the specified user is part of a group that are granted with these permissions (so it's not granted individually for the user) and it's also ok to have permissions on the cluster that propagate to all the vms within the cluster (so it's not individually per VM)

how about replacing this with:
```suggestion
*** `DiskCreator` and `UserVmManager` permissions for the virtual machines you want to update.
```
